tests/test_cli.py

```diff
@@ -21,60 +17,39 @@
     from typing import IO  # NOQA
     from typing import List  # NOQA
     from typing import Optional  # NOQA
-    from typing import Tuple  # NOQA
     from typing import Type  # NOQA
 
-TEST_CONFIG_TEMPLATE = 'default_storage: sqlite:///{default_storage}\n'
+    from optuna.trial import Trial  # NOQA
 
 
-class StorageConfigSupplier(object):
-    def __init__(self, config_template):
-        # type: (str) -> None
+class StorageSupplier(object):
```

Comment:
Since we no longer need `tempfile_config`, can't we delete `StorageSupplier` in this file and use `optuna.testing.storage.StorageSupplier` instead?

For example, we could modify `test_create_study_command` as follows:
```python
def test_create_study_command():
    # type: () -> None

    with StorageSupplier('new') as storage:
        storage_url = str(storage.engine.url)

        # Create study.
        command = ['optuna', 'create-study', '--storage', storage_url]
        subprocess.check_call(command)

        # Command output should be in name string format (no-name + UUID).
        study_name = str(subprocess.check_output(command).decode().strip())
        name_re = r'^no-name-[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$'
        assert re.match(name_re, study_name) is not None

        # study_name should be stored in storage.
        study_id = storage.get_study_id_from_name(study_name)
        assert study_id == 2
```

Reply:
Thank you for pointing it out. I fixed it based on your example code in this commit https://github.com/pfnet/optuna/pull/653/commits/5b13c76da57feae59d409f556077d55fd9e48b78.

```diff
def test_create_study_command():
    # type: () -> None

    with StorageSupplier('new') as storage:
+       assert isinstance(storage, RDBStorage)
        storage_url = str(storage.engine.url)
```

I needed to add an assersion that confirms `storage` is an instance of `RDBStorage` because `optuna.testing.storages.StorageSupplier` can returns `None` or `InMemoryStorage`.